styles.css

```diff
@@ -64,7 +64,9 @@ div.videotimetab-resize {
 .modtype_videotime.label_mode,
 .modtype_videotime.label_mode .mod-indent-outer,
 .modtype_videotime.label_mode .contentwithoutlink {
-    display: block !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
This change was flagged by the code checker is this an accidental changed? @stefanscholz

Reply:
Added a space. @dthies